frontend/src/Components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './LoginForm.css'; 
+
+const LoginForm = () => {
+    return (
+        <div className='Starter'>
+            <form action=''>
+                <h1>Login</h1>
+                <div className='input-box'>
+                    <input type="text" placeholder='Username' required />
+                </div>
+                <div className='input-box'>
+                    <input type="password" placeholder='Password' required />
+                </div>
+
+                <div className="remember-forgot">
+                    <label><input type="checkbox" />Remember me</label>
```

Comment:
nit: the input itself generally lives outside as a sibling to the label, rather than as a child! let's de-nest these

Reply:
Just did! Lmk if I did it to you're liking